provisioner/docker/docker-hadoop.sh

```diff
@@ -215,7 +215,11 @@ destroy() {
         echo "No cluster exists!"
     else
         get_nodes
-        docker exec ${NODES[0]} bash -c "umount /etc/hosts; rm -f /etc/hosts"
+        if [ -z ${NODES} ]; then
+          echo "No cluster running, but provision id is exists."
```

Comment:
How about `"No cluster is running but .provision_id exists."` ? @yamasakisua

Reply:
I see.  I fixed it.